### PR TITLE
iDecayV2

### PR DESCRIPTION
**Pull request by Quick-Flash**
Replaces the current iDecay. Just changes the strength of idecay as iterm approaches 0. Removes some bobbles and other issues that iDecay seem to present. This can be made almost the same as iDecay if the cutoff is set to 1. Tester reports are that this is much better.

### CODE REVIEW

**Comment by nerdCopter** [comment]
* Test and merge #321 first.
* Menu will require mods upon updating branch.

**Comment by gretel** [comment]
this has been reported repeatdely since october last year.

> Removes some bobbles and other issues that iDecay seem to present

thanks for fixing one of the longest standing and most ignored bugs since 0.1.0.

**Comment by Quick-Flash** [comment]
> this has been reported repeatedly since october last year.
> 
> > Removes some bobbles and other issues that iDecay seem to present
> 
> thanks for fixing one of the longest standing and most ignored bugs since 0.1.0.

yeah, recently figured out fully what the source of some of it was. trouble shooting can be a pain. Took us a few solid weeks of trying to find the culprit.

**Comment by Quick-Flash** [comment]
Good to merge now.

**Comment by nerdCopter** [comment]
* i updated the branch, but had to PR the fix separately.
* making this note for records
* `src/main/cms/cms_menu_imu.c` : `i_decay_cutoff = pidProfile->i_decay_cutoff;`

File: src/main/cms/cms_menu_imu.c [hunk 1]
```diff
@@ -222,12 +228,6 @@ static long cmsx_PidWriteback(const OSD_Entry *self)
         pidProfile->pid[i].I = tempPid[i][1];
         pidProfile->pid[i].D = tempPid[i][2];
     }
-    pidProfile->feathered_pids = feathered_pids;
```
Comment (nerdCopter) [inline src/main/cms/cms_menu_imu.c:225]: i forgot to remove these in my `20200714_advanced_menu` branch

**Pull request merged**